src/lib/rest/rest.cpp

```diff
@@ -1168,6 +1182,8 @@ static int restStart(IpVersion ipVersion, const char* httpsKey = NULL, const cha
                                    MHD_OPTION_HTTPS_MEM_KEY,            httpsKey,
                                    MHD_OPTION_HTTPS_MEM_CERT,           httpsCertificate,
                                    MHD_OPTION_CONNECTION_MEMORY_LIMIT,  memoryLimit,
+                                   MHD_OPTION_CONNECTION_LIMIT,         maxConns,
+                                   MHD_OPTION_THREAD_POOL_SIZE,         threadPoolSize,
```

Comment:
Nope, it would give an error (if pool size > 1).
However, this never happens as:

    if (threadPoolSize != 0)
    {
      serverMode = MHD_USE_SELECT_INTERNALLY;
      ...

Tested empirically. It would be nice to have a make step to run all functests with/without MHD thread pool.
I have done this, once, manually but it wouldn't hurt to have it automatized ...